src/lib/RpcApi.ts

```diff
@@ -147,17 +147,22 @@ export default class RpcApi {
                     hasRequest: !!this._staticStore.request,
                 });
 
+                let account;
+                if (request && 'walletId' in request) {
```

Comment:
As I noticed in my PR (https://github.com/nimiq/accounts/pull/197/files#diff-5354875441714f08e8db0466207385fdL157), the check for `walletId in request` also returns true when `request.walletId` is `undefined` (which happens when the RequestParser does `request.walletId = publicRequest.accountId`. If `accountId` does not exist, `walletId` gets set on the request object, but as `undefined`. I thus propose to change this check in the same way as in the above linked diff.

Reply:
Alright :+1: